### ISSUE TITLE
Missing local build/installation instructions

### ISSUE THREAD

**Issue by marad** [kwebio/kweb-core#126]
I want to test my changes locally so I made changes, updated version `build.gradle` to `0.7.9-SNAPSHOT` and ran `./gradlew install`.

The package is installed but it's missing some classes:

```kotlin
import kweb.plugins.fomanticUI.fomanticUIPlugin
import kweb.plugins.staticFiles.StaticFilesPlugin
import kweb.state.KVar
import kweb.state.render
```

Is there any special build/install step I'm missing? Maybe it would help to gather such instructions in some `contributors.md` or something like that?

**Comment by marad** [comment]
Ok, I've discovered that in build.gradle, the artifact group was set to 'kweb' instead of 'com.gibhut.kwebio'. Why was that? Is that intentional?

**Comment by sanity** [comment]
Hmm, that looks like a mistake, although I'm not sure why it didn't cause issues previously.  Fixed in `master` branch.

**Comment by marad** [comment]
I see that it's fixed now so closing I'm closing this issue.

**Comment by sanity** [comment]
Thanks, I've just released [0.7.9](https://github.com/kwebio/kweb-core/releases/tag/0.7.9) with this fix.
